Content.Shared/Follower/FollowerSystem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Shared.Database;
+using Content.Shared.Ghost;
+using Content.Shared.Movement.EntitySystems;
+using Content.Shared.Verbs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Robust.Shared.Log;
+
+namespace Content.Shared.Follower;
+
+public class FollowerSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<GetAlternativeVerbsEvent>(OnGetAlternativeVerbs);
+        SubscribeLocalEvent<FollowerComponent, RelayMoveInputEvent>(OnFollowerMove);
+    }
+
+    private void OnGetAlternativeVerbs(GetAlternativeVerbsEvent ev)
+    {
+        if (!HasComp<SharedGhostComponent>(ev.User))
+            return;
+
+        var verb = new Verb
+        {
+            Priority = 10,
+            Act = (() =>
+            {
+                var follower = EnsureComp<FollowerComponent>(ev.User);
+                follower.Following = ev.Target;
+                var xform = Transform(ev.User);
+                xform.Coordinates = Transform(ev.Target).Coordinates;
+                xform.AttachParent(ev.Target);
```

Comment:
So, there's one more problem with AttachParent is that if the followed entity gets deleted then so does the ghost. I tested it on Saltern and Packedstation and they spawned a new ghost at what seems like the observer spawn point.